packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -47,7 +47,7 @@ export interface ISpinButtonState {
 }
 
 export type DefaultProps = Required<
-  Pick<ISpinButtonProps, 'step' | 'min' | 'max' | 'disabled' | 'labelPosition' | 'label' | 'incrementButtonIcon' | 'decrementButtonIcon'>
```

Comment:
So we're not actually removing min & max as props just from the default props? Also I checked out the issue - do we know why this was causing the max to be 10? Seems random

Reply:
I am removing their default values, but not removing these as props. We were setting default values:
![image](https://user-images.githubusercontent.com/1207059/70173787-7440f980-1688-11ea-871c-a606af8a0cda.png)